src/Model/Union.php

```diff
@@ -0,0 +1,462 @@
+<?php
+
+declare(strict_types=1);
+
+namespace atk4\data\Model;
+
+use atk4\data\Exception;
+use atk4\data\Field;
+use atk4\data\FieldSqlExpression;
+use atk4\data\Model;
+use atk4\dsql\Expression;
+use atk4\dsql\Query;
+
+/**
+ * Union model combines multiple nested models through a UNION in order to retrieve
```

Comment:
usecase, what is best alternative without it? We should select table name (or something unique per model, like it's object hash) and make the result rw, right?